src/components/events/event-location.tsx

```
@@ -0,0 +1,57 @@
+import React from 'react';
+import { Heading3 } from '../ui/heading-3';
+
+type EventLocationProps = {
+  latitude?: number | null;
+  longitude?: number | null;
+  city: string;
+  address: String;
```

Biome (1.9.4)

[error] Don't use 'String' as a type.

Use lowercase primitives for consistency.
Safe fix: Use 'string' instead


(lint/complexity/noBannedTypes)